build.gradle

```diff
@@ -133,7 +133,10 @@ allprojects {
     }
   }
 
-  dependencies { errorprone("com.google.errorprone:error_prone_core") }
+  dependencies {
+    compile fileTree(dir: "${rootDir}/libs", include: "*.jar")
```

Comment:
Only a runtime dependency, so I think ` runtime fileTree(dir: 'libs', include: '*.jar')` is more appropriate. I added it to at line 435 but I don't think it makes a difference.